Makefile

```diff
@@ -3,7 +3,7 @@ MODULE_NAME ?= keda
 # Semantic Module Version used for identifying the build
 MODULE_VERSION ?= 0.0.4
 # Module Registry used for pushing the image
-MODULE_REGISTRY_PORT ?= 50092
+MODULE_REGISTRY_PORT ?= 63368
```

Comment:
why the new port?

Reply:
As we agreed offline we will set it to `8888`